docs/DeveloperGuide.md

```diff
@@ -834,3 +880,21 @@ testers are expected to do more *exploratory* testing.
        The terminal from where `WedLinker.jar` is launched should log where the file is corrupted.
 
     3. If the data is beyond repair, delete the entire `docs` folder and the `AddressBook.json` file to start afresh.
+
+
+## Appendix: Planned Enhancements
+
+Team size: 5
+
+Based on the current implementation of WedLinker, there are known bugs and limitations that we are unable to resolve due to
+feature freeze. The plans to improve our features are as such.
+
+1. There is a small GUI bug that causes the overflow of names for Tags and Weddings which causes cosmetic flaws.
+Planned enhancements would be to truncate the name with `...`.
+2. There is a duplicate validation bug that allows the creation of duplicate Person, Wedding, Tag, Task, Tags.
```

Comment:
Maybe can elaborate more here on what exactly is meant by duplicates because exact duplicates are detected, just not ones with small variations (I know you say this below in the planned enhancements implicitly but just to be sure)